### PR TITLE
Implement slider widget for WebUI

### PR DESCRIPTION
**Pull request by resetnow**
This commit implements slider widget in Classic UI. 

Changes overview:
* `slider.html`: the snippet was modified and now contains slider elements along with the initialization parameters
* `Logic.js`: contains slider code. Added `sliderchange` callback.
* `main.html`: processes value change events and sends requests to OpenHAB.
* `Render.css`: slider stylesheet

Tested with recent Firefox and Chrome versions, Android 5.1 stock browser and Chrome for Android. I don't have any Apple devices, so it would be great if someone could test it with iphone/ipad.

Here is how it looks like: [Desktop](http://i.imgur.com/Lu9Li0u.png), [Android](http://i.imgur.com/dVNEdQe.png).

References:  #2474

### CODE REVIEW

**Comment by resetnow** [comment]
Regarding `Logic.min.js

**Comment by teichsta** [comment]
Thanks @resetnow, this looks pretty nice already! @sja could you please have a look at this one?

**Comment by udo1toni** [comment]
still work in progress?

**Comment by teichsta** [comment]
good question … @sja?

**Comment by sja** [comment]
@resetnow You're right, its not so nice to check .min.js in, but here we don't have any infrastructure to generate this on build time. So I chose the simple way.
For OH2, we use the frontend-maven plugin to dynamically set up the nodejs/grunt/bower stack.

@udo1toni Sorry for the delay, thanks for the reminder. I was getting confused by the other issues around ClassicUI. Will answer this PR today.

File: bundles/ui/org.openhab.ui.webapp/web/WebApp/Action/Logic.js [hunk 1]
```diff
@@ -1713,7 +1731,151 @@ var WebApp = (function() {
...
+		_t.opt = item.parentNode;
+		_t.progress = item.querySelector(".iSliderProgress");
+		_t.handle = item.querySelector(".iSlider");
+		_t.valueEl = _t.opt.parentNode.querySelectorAll("span")[4];
+		_t.max = 100;
+		_t.min = 0;
+		_t.dragState = undefined;
+		_t.interval = undefined;
+		_t.computedOffset = 
+			parseInt(_style.getPropertyValue("border-top-left-radius")) * 2 +
```
Comment (sja) [inline bundles/ui/org.openhab.ui.webapp/web/WebApp/Action/Logic.js]: Style: Missing radix

File: bundles/ui/org.openhab.ui.webapp/web/WebApp/Action/Logic.js [hunk 2]
```diff
@@ -1713,7 +1731,151 @@ var WebApp = (function() {
 			}
 		}
 	}
-
+	
+	function Slider(item) {
+		var _t = this;
```
Comment (sja) [inline bundles/ui/org.openhab.ui.webapp/web/WebApp/Action/Logic.js]: Style: Well, yes, this scripts var names look like so, but wouldn't it be nice to start new functions with more expressive var names? Especially when it's getting minified, it wouldn't make a difference.

**Comment by sja** [comment]
Tested and looks good to me. Thanks to @resetnow for this contribution. I think that will help new users a lot to understand that there is now a real slider.

File: bundles/ui/org.openhab.ui.webapp/web/WebApp/Action/Logic.js (hunk 2 above)
Comment (resetnow) [inline bundles/ui/org.openhab.ui.webapp/web/WebApp/Action/Logic.js]: I just wanted something to refer to instance of Slider in nested scopes. Sometimes `that` is used as a variable name. Should I change it to `that` or something other?

Comment (sja) [inline bundles/ui/org.openhab.ui.webapp/web/WebApp/Action/Logic.js]: Maybe `that`. The underscore confused me at first sight. `(function(a, t)...` is another example. And some comments for functions would be nice. But you're right, the rest of `Slider` is much more readable that the original Logic.js.

**Comment by resetnow** [comment]
The problems mentioned here were fixed.

**Comment by resetnow** [comment]
If there aren't any other problems, I suppose it can be merged then?

**Comment by resetnow** [comment]
Shame on me! I must have been testing a cached version...

**Comment by resetnow** [comment]
Just a note: previously mentioned issues are fixed and it works...

**Comment by teichsta** [comment]
Thanks for the update, guys! Since you have implemented the Basic-UI for ESH meanwhile i assume this PR won't make it into ESH anymore, right?

**Comment by teichsta** [comment]
Thanks @resetnow!

**Comment by resetnow** [comment]
>Since you have implemented the Basic-UI for ESH meanwhile i assume this PR won't make it into ESH anymore, right?

Yes, unless ESH maintainers would like to see this ported. Personally I think it's good as it is, there is a new ui anyway — although some fixes still need to be ported to ESH (like colorpicker constant update).

**Comment by kaikreuzer** [comment]
> Yes, unless ESH maintainers would like to see this ported. Personally I think it's good as it is, there is a new ui anyway

No! This would mean a regression once openHAB 1 users switch to openHAB 2. The Classic UI will continue to exist for the moment, so all fixes must be ported as well!

**Comment by resetnow** [comment]
@kaikreuzer ok, I'll port it then, probably this weekend.

**Comment by resetnow** [comment]
@kaikreuzer, a question: ClassicUI in Eclipse Smarthome contains a minified Logic.js version. Openhab, however, has unminified original version introduced in https://github.com/openhab/openhab/commit/a5a77baf358a01459668af9e69cf3a4394d12b55. Should the uncompressed version (and minify script) be ported by @sja or can I just copy Logic.js from Openhab?

**Comment by kaikreuzer** [comment]
Probably a question that @sja should answer - I wasn't actually aware that we have minified in ESH and non-minified in openHAB. I would prefer if it is done in the same way...

**Comment by resetnow** [comment]
@sja so can you please copy this commit to ESH or do you mind if I copy it?

**Comment by sja** [comment]
The latest and greatest was done for OH1, so yes, there is a gap.

On adding the minifiy script, I also did a small change in Logic.js. The minified version Logic.min.js was there from the beginning. I added the unminified version to be able to make changes and minify it again. Because I don't know the current state of the OH2 Logic(.min).js. I will sync them.

**Pull request merged**